### ISSUE TITLE
Getting Protocol error (Page.createIsolatedWorld): 'Page.createIsolatedWorld' wasn't found error on Jenkins

### ISSUE THREAD

**Issue by m2kannojiya** [GoogleChrome/lighthouse#4460]
In my local machine (windows) I am able to get Accessibility report but on Jenkins I am getting below error:
Audit error: Required Accessibility gatherer encountered an error: Protocol error (Page.createIsolatedWorld): 'Page.createIsolatedWorld' wasn't found
for each accessibility property. What could be the reason?
![accebilityissue-jenkins](https://user-images.githubusercontent.com/3115182/35954192-4f16788c-0cae-11e8-9fd7-299e9e02cf8b.PNG)

**Comment by patrickhulce** [comment]
@m2kannojiya how old is your chrome version on Jenkins? Is it allowed to auto-update?

**Comment by m2kannojiya** [comment]
@patrickhulce I am using lighthouse-cli and chrome-flags set for --headless. Is it still need chrome browser in machine?
How can we check  if it is allowed to auto-update?

**Comment by m2kannojiya** [comment]
@patrickhulce chrome driver version is 1.2

**Comment by patrickhulce** [comment]
>  Is it still need chrome browser in machine?

Yes, Chrome needs to be installed on the machine in order for Lighthouse to run.

> chrome driver version is 1.2

This sounds very old, the latest version is [2.35](https://sites.google.com/a/chromium.org/chromedriver/downloads). You'll need to update Chrome to latest in order for Lighthouse to work. I'm converting this issue to a feature that would detect your Chrome version early and give you a clearer warning, but it's not really a LH bug per se :)

**Comment by m2kannojiya** [comment]
@patrickhulce you were right after updating to latest driver it's working fine. Thanks for help
